src/nvim/ex_cmds.lua

```diff
@@ -614,7 +614,7 @@ return {
   },
   {
     command='cquit',
-    flags=bit.bor(TRLBAR, BANG),
+    flags=bit.bor(TRLBAR, BANG, WORD1),
```

Comment:
It is more consistent with COUNT plus NOTADR and not WORD1 and argument: this would allow typing `:2cq<CR>` in addition to `:cq2<CR>` (which is consistent with e.g. `:cnext`), and simplify `ex_cquit` to a great extent: you just call `getout(eap->addr_count > 0 ? eap->line2 : 1)`.